Sources/RDeviceIdentifier/RDeviceIdentifierConstants.swift

```diff
@@ -0,0 +1,7 @@
+enum RDeviceIdentifierConstants {
+    // Because "jp.co.rakuten.ios.sdk.deviceinformation" is using as a keychain-access-group key, so we keep using this key.
+    static let keychainAccessGroup = "jp.co.rakuten.ios.sdk.deviceinformation"
```

Comment:
I don't think we should have that in a public repo.

It should be injected into the initializer.

Reply:
For now, it's fine to change this to "shared.keychain.access.group".

I don't want that sensitive information committed to master.